src/encoded/static/components/matrix_experiment.js

```diff
@@ -292,12 +292,21 @@ const MatrixHeader = ({ context }) => {
         }
     }
 
+    // If the user has requested an ENCORE matrix, generate a matrix description.
+    const query = new QueryString(context.search_base);
+    const matrixDescription = query.getKeyValues('internal_tags').includes('ENCORE') ?
```

Comment:
Will ENCORE ever be lower case?

Reply:
That's a good point, but through our menu it could never be lower case, and Ben has said if the user modifies the URL, they’re on their own.